### PR TITLE
New package: wabt-1.0.19

### PR DESCRIPTION
**Pull request by ifreund**

### CODE REVIEW

File: srcpkgs/wabt/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+homepage="https://github.com/WebAssembly/wabt"
+_wasm_c_api_commit=d9a80099d496b5cdba6f3fe8fc77586e0e505ddc
+distfiles="https://github.com/WebAssembly/wabt/archive/${version}.tar.gz
+ https://github.com/WebAssembly/wasm-c-api/archive/${_wasm_c_api_commit}.tar.gz"
+checksum="134f2afc8205d0a3ab89c5f0d424ff3823e9d2769c39d2235aa37eba7abc15ba
+ aea8cd095e9937f1e14f2c93e026317b197eb2345e7a817fe3932062eb7b792c"
+
+pre_configure() {
+	# copy submodule to proper location
+	cp -rup ${XBPS_BUILDDIR}/wasm-c-api-*/* ${wrksrc}/third_party/wasm-c-api
```
Comment (Chocimier) [inline srcpkgs/wabt/template:18]: Please rather move like in `tracebox`, otherwise ${XBPS_BUILDDIR}/wasm-c-api won't be cleaned.

Comment (ifreund) [reply srcpkgs/wabt/template:18]: Will fix for this and `wasmtime`, I just copied blindly from `ppsspp` which I will also fix in a minute then.

**Comment by Chocimier** [comment]
Please add `make_check_target=check`, `checkdepends="python3-ply"` and gtest, testsuite submodules so tests can be run. Use d2163dace09d647bccf34b9b82a6f05a3b23cf29 version of testsuite, matching release.

On moving submodules, use commit hashes rather than globbing:
```
rmdir third_party/wasm-c-api
mv ../wasm-c-api-${_wasm_c_api_commit} third_party/wasm-c-api
```

**Comment by ifreund** [comment]
@Chocimier Thanks for the review, made the suggested changes. Needed to add the testsuite submodule as well to get the tests passing.

**Pull request merged**